resources/gobos/glass-black-10-circles.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/gobo.json",
+  "name": "Black with 10 circles",
```

Comment:
"Black" doesn't make sense, since the resulting light beams are completely blocked by the black parts. Maybe just `10 circles`?